src/katex.less

```diff
@@ -547,6 +547,22 @@
         border: 0.04em solid black;   // \fboxrule = 0.4pt
     }
 
+    .actangle-pad {
+        padding-right: @muspace * 2; //2mu
+        padding-left: @muspace;
```

Comment:
We actually don't use `@muspace` nor any of the other "space" constants in this file.  I'd like to actually get rid of `@muspace` et al.  Could you create an internal `@actangle` command that renders the symbol without the padding and then define a macro for the actual command that includes the the padding/margin using `\mkern`?